### PR TITLE
Chore(docs) remove section of Reviewing docs to  be moved into PR guide

### LINKED ISSUE #10: Reviewing
- [x] Move content to the PR guidelines - https://github.com/mdn/temp-processes/pull/4
- [x] Keep the markdown file and link to pr guidelines, add specifics per repo - https://github.com/mdn/project-template/pull/11

### PR DESCRIPTION
**Pull request by bsmth**
Fixes #10 

__Notes:__
After the [linked pull request](https://github.com/mdn/temp-processes/pull/4) is merged, the temp-processes repo content will be incorporated into MDN Web Docs (pending other changes).

__Related pull requests:__

- [ ] Moved content into https://github.com/mdn/temp-processes/pull/4

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Community Participation Guidelines
```
Comment (dipikabh) [inline CODE_OF_CONDUCT.md]:
```suggestion
# Community participation guidelines
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 
 This repository is governed by Mozilla's code of conduct and etiquette guidelines.
-For more details, please read the
-[Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
+For more details, please read the [Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
 
 ## How to Report
```
Comment (dipikabh) [inline CODE_OF_CONDUCT.md]:
```suggestion
## Reporting violations
```

File: REVIEWING.md [hunk 3]
```diff
@@ -1,99 +1,21 @@
...
-  - [Values](#values)
-  - [Process](#process)
-    - [Maintainer edge cases](#maintainer-edge-cases)
-    - [General review guidance](#general-review-guidance)
-  - [Checklist](#checklist)
-  - [Reading List](#reading-list)
+## Values and guidelines
 
-## The Reviewer Role
+All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md) and are also protected by it.
```
Comment (dipikabh) [inline REVIEWING.md]:
```suggestion
All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md); they are also protected by the Code of Conduct.
```

File: REVIEWING.md [hunk 4]
```diff
@@ -1,99 +1,21 @@
...
+A reviewer should not tolerate poor behavior and is encouraged to [report any behavior](CODE_OF_CONDUCT.md#How_to_Report) that violates the Code of Conduct.
```
Comment (dipikabh) [inline REVIEWING.md]: This link might change based on my title suggestion in the other file.

How about extending the link to:
```suggestion
A reviewer should not tolerate poor behavior and is encouraged to [report any behavior that violates](CODE_OF_CONDUCT.md#How_to_Report) the Code of Conduct.
```

File: REVIEWING.md [hunk 5]
```diff
@@ -1,99 +1,21 @@
...
 
-The reviewer role is distinct from the maintainer role. Reviewers can approve a pull request but they cannot merge it. A maintainer handles final approval and merging the pull request.
+Everyone participating must follow our [Community Participation Guidelines](CODE_OF_CONDUCT.md#Community_Participation_Guidelines)
 
-## Values
+## Code review process
```
Comment (dipikabh) [inline REVIEWING.md]:
```suggestion
## Review process
```
..to not be taken literally as code 🙂

File: REVIEWING.md [hunk 6]
```diff
@@ -1,99 +1,21 @@
...
 
-All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md) and are also protected by it. A reviewer should not tolerate poor behavior and is encouraged to [report](CODE_OF_CONDUCT.md#enforcement) any behavior that violates the Code of Conduct. All of our values listed below are distilled from our Code of Conduct.
+MDN Web Docs have a well-defined review process which must be followed by reviewers in all repositories under this organization.
```
Comment (dipikabh) [inline REVIEWING.md]:
```suggestion
The MDN Web Docs team has a well-defined review process that must be followed by reviewers in all repositories under the GitHub MDN organization.
```
makes sense?

File: REVIEWING.md [hunk 7]
```diff
@@ -1,99 +1,21 @@
...
+This process is described in the [pull requests](https://developer.mozilla.org/en-US/docs/MDN/Community/Pull_requests) documentation page.
```
Comment (dipikabh) [inline REVIEWING.md]:
```suggestion
This process is described in detail on the [Pull request guidelines](https://developer.mozilla.org/en-US/docs/MDN/Community/Pull_requests) page.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Some suggestions for your consideration

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (bsmth) [reply CODE_OF_CONDUCT.md]: Very good, tnx

File: REVIEWING.md (hunk 4 above)
Comment (bsmth) [reply REVIEWING.md]: Good, have updated to match the new anchor 👍🏻

**Comment by bsmth** [comment]
Thanks, @dipikabh, I've added changes based on your feedback if you want to have a quick look 👀

**Review by dipikabh — APPROVED** [review]
Looks good. Thank you! 👍

**Pull request merged**